README.md

```diff
@@ -102,7 +99,7 @@ npm run start
 
 ## Screenshot of the page
 
-![image](https://user-images.githubusercontent.com/60612329/136966291-3d66b74a-a1e5-44a9-85e5-dadf8d1e3c47.png)
+![image](https://user-images.githubusercontent.com/60612329/137308844-e2dcb363-d40e-4c6c-82e7-7baf5daae584.png)
```

Comment:
Your app isn't loadinging information from API check and fix this error running the project locally:
<img width="1440" alt="Screen Shot 2021-10-14 at 6 43 53" src="https://user-images.githubusercontent.com/22258274/137311176-18b123c9-f31d-478a-9b19-4cf2faef3b0a.png">

Reply:
it would be really helpful @carlos-ssh

Reply:
> Thank you for the updates but I don't understand??
> it works fine on my side
> 
> ![image](https://user-images.githubusercontent.com/60612329/137313283-4b46394c-0332-415f-9e0b-aa1a253ad75e.png)
> 
> if I can have more details about the error

Error 400 the requests isn't connected to the API. Try checking your API key or instead check if you have changes locally push a commit to GH.